piecrust/src/session.rs

```diff
@@ -762,6 +786,7 @@ struct Deploy {
     module_id: ModuleId,
     bytecode: Vec<u8>,
     fdata: Option<Vec<u8>>,
+    owner: [u8; 32],
```

Comment:
Maybe we should include the whole `DeployData` struct here instead of `owner`, `fdata`, and `module_id` separately.

Reply:
Problem is that DeployData has a type argument `DeployData<Arg>` and this leads to a ripple effect leading to code that does not have the `Arg` available. This will require major restructuring and adding the `Arg` in lots of places. I am not sure it is worth it.

Reply:
You're right here. Not worth it.